### ISSUE TITLE
End Day Button

### ISSUE THREAD

**Issue by DanielaGarza** [MD2Korg/mCerebrum-Study#17]
Date: 4/5/2016
Time: 4:00 PM
2 phones (DG 4 and PF 4) do not have an "End Day" button and cannot end the day

**Comment by monowar** [comment]
It will show after 8 hours from the day started. After clicking day end, you will not get any EMA after that (assuming that the participant is completed the data collection and don't want to get any EMA after this). 

Is this okay for your study?

**Comment by DanielaGarza** [comment]
Oh I see that makes sense. Yes, I think that is ok for the study. If we need anything changed, we will let you know for full study.

**Comment by DanielaGarza** [comment]
Hi Monowar,

 

Actually, Paty mentioned that her phone didn’t get an “end day” button all day. 

 

Her EMA Schedule app isnt’ functional based on the instructions you gave me to look for EMA triggers and conditions in the app.

**Comment by monowar** [comment]
@DanielaGarza I fixed "day end" problem. the "End Day" button will be visible after 8 hours from the day started. 
I am closing this issue assuming that, it is solved. If you see this problem again, please let me know, or reopen this issue.

**Comment by DanielaGarza** [comment]
Date: 4/7/2016
Time: 7:49 AM
I started my day yesterday at 5:30 PM - because I went to sleep at 10 PM, I did not see the "Day Ended" button before bed so the day did not end. This morning I pressed "Day Ended" and now I do not have a "Day Started" button so I cannot start my day. Could we set it such that day ends if the devices are charging? or if data quality "not good" and the time is past "sleep time"?
